diff --git a/rrrspec-web/lib/rrrspec/web/app.rb b/rrrspec-web/lib/rrrspec/web/app.rb
index 5e82786..9f275d2 100644
--- a/rrrspec-web/lib/rrrspec/web/app.rb
+++ b/rrrspec-web/lib/rrrspec/web/app.rb
@@ -20,6 +20,10 @@ module RRRSpec
         set :haml, :format => :html5
       end
 
+      after do
+        ActiveRecord::Base.connection.close
+      end
+
       get('/') { haml :index }
       get('/tasksets/*') { haml :taskset }
     end
